doc/how_to_guides/isaac_panda/.docker/Dockerfile

```diff
@@ -14,15 +14,31 @@ RUN apt-get update && apt-get upgrade -y && rosdep update
 # Install packages required to run the demo
 RUN apt-get install -y --no-install-recommends \
     ros-humble-moveit \
-    ros-humble-moveit-resources \
-    ros-humble-moveit2-tutorials
+    ros-humble-moveit-resources
+#    ros-humble-moveit2-tutorials
```

Comment:
@tylerjw this didn't work :( Any ideas?

I get the following error when building the container.
```
------                                                                                                   
 > [ 5/16] RUN apt-get install -y --no-install-recommends     ros-humble-moveit     ros-humble-moveit-resources     ros-humble-moveit2-tutorials:                                                                 
#0 0.409 Reading package lists...
#0 0.869 Building dependency tree...
#0 0.970 Reading state information...
#0 0.977 E: Unable to locate package ros-humble-moveit2-tutorials
------
```